add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
**Observations**: The current file structure and code organization do not adhere to modern PHP architectural patterns, leading to potential difficulties in scaling and testing.

**Recommendations**: Consider implementing a Model-View-Controller (MVC) architecture. Use a service or repository pattern for database interactions and a controller for handling requests. This will make the application more modular and testable.

**Example**:
Create a `BookRepository` class to handle database operations:
```php
class BookRepository {
    private $connection;

    public function __construct($connection) {
        $this->connection = $connection;
    }

    public function addBook($title, $author, $year) {
        $query = "INSERT INTO books (title, author, year) VALUES (?, ?, ?)";
        $stmt = mysqli_prepare($this->connection, $query);
        mysqli_stmt_bind_param($stmt, 'ssi', $title, $author, $year);
        return mysqli_stmt_execute($stmt);
    }
}
```
Then use this repository in your controller logic.